src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/ExpenseRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+
+public interface ExpenseRepository extends JpaRepository<Expense,Long> {
```

Comment:
Although is not needed it's good practice to annotate repositories with @Repository annotation